src/main/Config.cpp

```diff
@@ -1467,6 +1469,11 @@ Config::processConfig(std::shared_ptr<cpptoml::table> t)
                  [&]() { INVARIANT_CHECKS = readArray<std::string>(item); }},
                 {"INVARIANT_EXTRA_CHECKS",
                  [&]() { INVARIANT_EXTRA_CHECKS = readBool(item); }},
+                {"STATE_SNAPSHOT_INVARIANT_LEDGER_FREQUENCY",
+                 [&]() {
+                     STATE_SNAPSHOT_INVARIANT_LEDGER_FREQUENCY =
```

Comment:
Looks like we're missing IS_VALIDATOR check below to ensure the new invariant can only be configured on watchers.